fineract-client/src/main/java/org/apache/fineract/client/util/FineractClient.java

```diff
@@ -26,7 +26,7 @@
  *
  * @author Michael Vorburger.ch
  */
-public class FineractClient {
+public final class FineractClient {
```

Comment:
.. but this shouldn't be required? Especially since in #1433 I'm making `createService()` a protected method, so this class should remain _open_ (extends-ible).